sdk/mobilenetwork/Azure.ResourceManager.MobileNetwork/src/Generated/Models/UeInfo4G.cs

```diff
@@ -0,0 +1,189 @@
+        /// <param name="mTmsi"> MME Temporary Mobile Subscriber Identity. </param>
+        /// <param name="groupId"> MME group identifier. </param>
+        /// <param name="code"> MME code. </param>
+        /// <param name="mccInfoGutiPlmnMcc"> Mobile country code (MCC). </param>
+        /// <param name="mncInfoGutiPlmnMnc"> Mobile network code (MNC). </param>
+        internal UeInfo4G(RatType ratType, DateTimeOffset? lastReadOn, IDictionary<string, BinaryData> serializedAdditionalRawData, string imsi, string imei, string imeisv, IList<UeSessionInfo4G> sessionInfo, string perUeTnla, int? mmeS1ApId, int? enbS1ApId, string lastVisitedTai, UeState? ueState, RrcEstablishmentCause? rrcEstablishmentCause, UeUsageSetting? ueUsageSetting, DateTimeOffset? lastActivityOn, string ngeNbId, string eNbId, string n3IwfId, string wagfId, string tngfId, string nid, int? bitLength, string gnbValue, string mccInfoConnectionInfoGlobalRanNodeIdPlmnIdMcc, string mncInfoConnectionInfoGlobalRanNodeIdPlmnIdMnc, string locationType, string tac, string mccInfoConnectionInfoLocationInfoPlmnMcc, string mncInfoConnectionInfoLocationInfoPlmnMnc, int mTmsi, int groupId, int code, string mccInfoGutiPlmnMcc, string mncInfoGutiPlmnMnc) : base(ratType, lastReadOn, serializedAdditionalRawData)
+        {
+            Imsi = imsi;
+            Imei = imei;
+            Imeisv = imeisv;
+            SessionInfo = sessionInfo;
+            PerUeTnla = perUeTnla;
+            MmeS1ApId = mmeS1ApId;
+            EnbS1ApId = enbS1ApId;
+            LastVisitedTai = lastVisitedTai;
+            UeState = ueState;
+            RrcEstablishmentCause = rrcEstablishmentCause;
+            UeUsageSetting = ueUsageSetting;
+            LastActivityOn = lastActivityOn;
+            NgeNbId = ngeNbId;
+            ENbId = eNbId;
+            N3IwfId = n3IwfId;
+            WagfId = wagfId;
+            TngfId = tngfId;
+            Nid = nid;
+            BitLength = bitLength;
+            GNBValue = gnbValue;
+            MccInfoConnectionInfoGlobalRanNodeIdPlmnIdMcc = mccInfoConnectionInfoGlobalRanNodeIdPlmnIdMcc;
+            MncInfoConnectionInfoGlobalRanNodeIdPlmnIdMnc = mncInfoConnectionInfoGlobalRanNodeIdPlmnIdMnc;
+            LocationType = locationType;
+            Tac = tac;
+            MccInfoConnectionInfoLocationInfoPlmnMcc = mccInfoConnectionInfoLocationInfoPlmnMcc;
+            MncInfoConnectionInfoLocationInfoPlmnMnc = mncInfoConnectionInfoLocationInfoPlmnMnc;
+            MTmsi = mTmsi;
+            GroupId = groupId;
+            Code = code;
+            MccInfoGutiPlmnMcc = mccInfoGutiPlmnMcc;
+            MncInfoGutiPlmnMnc = mncInfoGutiPlmnMnc;
+            RatType = ratType;
+        }
+
+        /// <summary> Initializes a new instance of <see cref="UeInfo4G"/> for deserialization. </summary>
+        internal UeInfo4G()
+        {
+        }
+
+        /// <summary> International mobile subscriber identifier. </summary>
+        public string Imsi { get; set; }
```

Comment:
From the description, looks like this property should be an identifier. Is `Imsi` a proper noun？

Reply:
Yes. As mentioned in summary it stands for International mobile subscriber identifier and widely used as "Imsi" only.